### DISCUSSION TITLE
Highlight new racecontrol messages with background

### DISCUSSION THREAD

**Discussion by stefanverkerknl** [f1multiviewer/issue-tracker#46] [Feature requests] [answered]
Is it possible to give new racecontrol messages in the Live Timings window a different background color (for example light blue) for a few seconds to highlight new messages so it draws some attention? 

Edit: maybe it's nice to have the same with drivers who climb up in the 'timing tower' in the Live Timings window. Drivers who climb up, give it a temporary light green background, drivers who lose a place, give them a light red background for 2 - 3 secs.

**Comment by stefanverkerknl** [comment]
To make it even more easier to see the places gained / lost, it would be great to have an option as well to have a fully vertical timing tower in the Live Timings window. See for example my Photoshopped idea:

![20220618-f1](https://user-images.githubusercontent.com/18143837/174406709-644d616d-8b6f-480a-953c-ec30988f7116.jpg)

**Reply by f1multiviewer** [reply]
This is possible now, though it requires popping out RCM

**Comment by f1multiviewer** [comment, marked as answer]
This is now done, although the color appears a bit shorter, but there's animations and all, and sounds, I think this is OK :)
